### ISSUE TITLE
How to use the workflow:  liveportrait_realtime_example_01?

### ISSUE THREAD

**Issue by PaiLI88** [kijai/ComfyUI-LivePortraitKJ#99]
Do you have a detailed tutorial? The camera reading seems to be a single frame, not a continuous recording. How should we film it specifically

**Comment by kijai** [comment]
> Do you have a detailed tutorial? The camera reading seems to be a single frame, not a continuous recording. How should we film it specifically

Yeah it's a single frame, but you can enable auto queue from the Comfy menu to simulate the realtime view:
![image](https://github.com/user-attachments/assets/6f20550a-3dee-4160-bb8e-2ebd24474f63)

**Comment by PaiLI88** [comment]
> > 你有详细的教程吗？摄像机读数似乎是单帧，而不是连续录制。具体应该怎么拍
> 
> 是的，这是一个单帧，但您可以从 Comfy 菜单中启用自动队列以模拟实时视图： ![图像](https://private-user-images.githubusercontent.com/40791699/352757167-6f20550a-3dee-4160-bb8e-2ebd24474f63.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIwOTU5OTgsIm5iZiI6MTcyMjA5NTY5OCwicGF0aCI6Ii80MDc5MTY5OS8zNTI3NTcxNjctNmYyMDU1MGEtM2RlZS00MTYwLWJiOGUtMmViZDI0NDc0ZjYzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI3VDE1NTQ1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY0MWQ3ZTkxNDM3ODA4NTFmOTE2ZGEwOTE4NDBjYzZiNmI5Zjg5YjJiNjc4ZGFiYjM2MTBmOTZmNGIyNDU3MTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5oVfdIb5RMZouxE_WdZ6XdvdYfd3nTGiHWBSJjosy2Q)

cool，Thank you very much for your reply. It was really helpful

**Comment by LittleAntChina** [comment]
Please tell me How do I get the local camera in the workflow: liveportrait_realtime_example_01?, I use the ivcam app phone camera as the local camera.
